xt/03_pod.t

```diff
@@ -1,3 +1,6 @@
+# XXX should be replaced by Dist::Zilla::Plugin::PodSyntaxTests, but holding
```

Comment:
What's wrong with Dist::Zilla::Plugin::PodSyntaxTests on Ubuntu 18.04?

Reply:
Unlike the current dependencies, it's not packaged, so it's a less convenient install.

I have spotted another problem, perhaps you can help? The META.json contains more data than META.yml, for example under `repository`. Any idea why?

Reply:
Packaged as in 'deb package'? I've never used distro Perl module packages as I can't chose the version or update it if needed.

Regarding META.json: I *think* that version 2 of the spec is the JSON file and the YAML is for backward compat, see https://metacpan.org/pod/CPAN::Meta::Spec#SERIALIZATION

Reply:
Thanks! I was worried there was an order problem in `dist.ini`.

Yes, deb packages. I've worked at a few places which have used the distribution's Perl and libraries. It has its pros and cons.

I'll merge this soon if you're okay with that?

Reply:
Sure, thanks for your work!